todoman/ui.py

```diff
@@ -224,7 +219,14 @@ def compact_multiple(self, todos):
             percent = todo.percent_complete or ''
             if percent:
                 percent = " ({}%)".format(percent)
-            urgent = " " if todo.priority in [None, 0] else "!"
+            if todo.priority == 5:
+                priority = "!!"
+            elif todo.priority == 4:
```

Comment:
So, will we have to raise an error also, if the priority does not belong to the range 0-9?